lib/Service/ObjectService.php

```diff
@@ -993,7 +1003,6 @@ private function addObject(
 
         // Setting inversedBy on sub object to parent currently will only work if the object is cascaded created for the first time!
         $subSchema = $this->getSchemaFromPropertyReference(property: $property, propertyName: $propertyName, schema: $schema);
-        $relations = $this->setInversedByRelation(item: $item, objectEntity: $objectEntity, subSchema: $subSchema, property: $property, index: $index);
```

Comment:
We remove this intentionally? (just checking)

Reply:
The function referenced does not exist